### ISSUE TITLE
During model conversion facing Keras layer of type <class 'tensorflow.python.keras.engine.training.Model'> is not supported.

### ISSUE THREAD

**Issue by VaneeshK** [edouardlp/Mask-RCNN-CoreML#15]
I used Conversions/task.py to convert the Keras .h5 file to .mlmodel but I am facing Keras layer of type <class 'tensorflow.python.keras.engine.training.Model'> is not supported. Any help is appreciated.
